tests/modules/datasets/test_share.py

```diff
@@ -1170,42 +1115,6 @@ def test_update_share_request_purpose_unauthorized(client, share1_draft, user, g
     assert 'UnauthorizedOperation' in update_share_request_purpose_response.errors[0].message
 
 
-def test_list_dataset_share_objects_approvers(client, user, group, share1_draft, dataset1):
```

Comment:
why do we remove these tests?

Reply:
Because the query `ListDatasetShareObjects` does not exist in the frontend, it is not used. We use the get_share_requests_to_me instead